File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        for (int num : nums) {
            int count = 0;

            for (int n : nums) {
                if (num == n) {
                    count += 1;
                }
            }
            if (count > 1) {
                return true;
            }
        }
        
        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}
        for i in nums:
            if i in freq:
                freq[i] += 1
            else:
                freq[i] = 1

        answer = []

        for x in range(k):
            most_common = max(freq, key=freq.get)
            answer.append(most_common)
            del freq[most_common]

        return answer